docs/project/faq.md

```diff
@@ -182,6 +197,67 @@ If a migration occurs, we anticipate:
     interoperability wrappers.
 -   Test-driven verification that migrations are correct.
 
+### Where should I ask questions about Carbon Language?
```

Comment:
Would this make sense under "What is Carbon's status?", right after the demo question?